components/gcp/automl/create_model_for_tables/component.py

```diff
@@ -24,11 +24,7 @@ def automl_create_model_for_tables(
     input_feature_column_paths: list = None,
     optimization_objective: str = 'MAXIMIZE_AU_PRC',
     train_budget_milli_node_hours: int = 1000,
-) -> NamedTuple('Outputs', [('model_path', str), ('model_id', str)]):
-    import sys
-    import subprocess
-    subprocess.run([sys.executable, '-m', 'pip', 'install', 'google-cloud-automl==0.4.0', '--quiet', '--no-warn-script-location'], env={'PIP_DISABLE_PIP_VERSION_CHECK': '1'}, check=True)
-
+) -> NamedTuple('Outputs', [('model_path', str), ('model_id', str), ('model_page_url', 'URI'),]):
```

Comment:
nit: not sure, is that an extra `,` at the end?

Reply:
Generally I put every output on its own line and have comma in the end so that new lines can be added without changing existing lines. (This is part of many style guides)
However here I had to put everything back in a single line due to a minor formatting issue when serializing yaml that was triggered by stripped hints leaving some trailing spaces.

This comma at the end of a list does not cause any problems in python.